### ISSUE TITLE
RFE: aws_vpc_peering_connection support for connection options

### ISSUE THREAD

**Issue by mblakele** [hashicorp/terraform#8169]
### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.
```
$ terraform -v
Terraform v0.7.0
```

### Affected Resource(s)
Please list the resources as a list, for example:
- aws_vpc_peering_connection

### Important Factoids
From https://www.terraform.io/docs/providers/aws/r/vpc_peering.html there doesn't appear to be a way to set `AllowDnsResolutionFromRemoteVpc` on an `aws_vpc_peering_connection` resource. This is a useful feature for various VPC configurations.

The aws cli equivalent is:

```
aws ec2 modify-vpc-peering-connection-options \
  --vpc-peering-connection-id "pcx-12345678" \
  --requester-peering-connection-options '{"AllowDnsResolutionFromRemoteVpc":true}' \
  --accepter-peering-connection-options '{"AllowDnsResolutionFromRemoteVpc":true}'
```

http://docs.aws.amazon.com/cli/latest/reference/ec2/modify-vpc-peering-connection-options.html

**Comment by kwilczynski** [comment]
@mblakele consider it done.

**Comment by stack72** [comment]
Closed via #8310
